### ISSUE TITLE
OTP verification shows after self registration screen. It should show before.

### ISSUE THREAD

**Issue by sencode** [covenantproject/cov-pat-mobileapp#8]
I tested today using covenant26042020.apk. Here are the findings:

1. I launched the app and entered my phone number.
2. The self registration screen showed up. I filled the form and pressed 'Register'
3. The OTP verification screen showed up. I entered the OTP that I got via SMS and logged in successfully.

Expected behavior:
The OTP verification screen should show right after I submit my phone number. Only after OTP verification is successful, I should be able to see the self registration screen. It used to work as expected in the previous version, but this new issue is found in the latest version (filename above).
